app/assets/stylesheets/custom.scss

```diff
@@ -47,6 +47,28 @@ h3 {
   min-height: 50px;
 }
 
+#flash_error {
+  text-align: center;
+  color: red;
+  font-weight: bold;
+  font-size: 18px;
+  margin-top: 1em;
+}
+
+#flash_alert {
+  color:red;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Colon after property should be followed by one space